### PR TITLE
Fix #1758: `VaSelect` actions and navigation

### LINKED ISSUE #1758: `VaSelect` keyboard navigation is completely broken
**Vuestic-ui version:**
`1.3.5` same in `1.4.0`

- [x] Fix all navigation buttons

**Steps to reproduce**
Open any `VaSelect` component and try to navigate or close it using keyboard buttons.

**What is the expected behavior?**
It should be fully usable just by keyboard.

**What is the current behavior?**
Nothing works - `esc`, `arrows`, `enter` etc.

**Other information**
Pretty sure that it was caused by the recent changes because it worked previously.

### PR DESCRIPTION
**Pull request by Derranion**
## Description
Close #1758

All `VaSelect` actions and keyboard navigation were completely broken, now it is fixed.

## Details:

- Keyboard navigation works
- UX improved - `VaSelect` will scroll to and focus last selected option
- All `actions` that were using `optionList` functions work now - `hintedSearch` etc.

</details>

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/ui/src/components/va-select/VaSelect.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@
     @keydown.up.stop.prevent="showDropdown()"
     @keydown.down.stop.prevent="showDropdown()"
     @keydown.space.stop.prevent="showDropdown()"
+    @keydown.enter.stop.prevent="showDropdown()"
```
Comment (RVitaly1978) [inline packages/ui/src/components/va-select/VaSelect.vue:19]: The native html `select` element has no such option. Do we want to do it?

**Review by RVitaly1978 — APPROVED** [review]

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/components/va-select/VaSelect.vue:19]: Native select option selection also works for `enter`.

**Review by aluarius — APPROVED** [review]

File: packages/ui/src/components/va-select/VaSelectOptionList/VaSelectOptionList.vue [hunk 2]
```diff
@@ -221,9 +221,12 @@ export default defineComponent({
       hoverNextOption,
       hoverFirstOption,
       focus,
+      scrollToOption,
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/VaSelectOptionList/VaSelectOptionList.vue:224]: Expose must work now :) after we fixed withTransportConfig.

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 1 above)
Comment (m0ksem) [reply packages/ui/src/components/va-select/VaSelect.vue:19]: Enter usually used for submit action in form. I would rather just use space to open select. Pressing enter on any component could submit my form.

Also, up and down buttons often used not for dropdown opening, but switching between options without opening dropdown.

**Review by m0ksem — APPROVED** [review]
So this PR fixes the bug. We can discuss correct component behavior later, in different task.

**Pull request merged**